homeassistant/components/whirlpool/sensor.py

```diff
@@ -119,11 +119,12 @@ class WhirlpoolSensorEntityDescription(
         key="DispenseLevel",
         name="Detergent Level",
         translation_key="whirlpool_tank",
+        entity_registry_enabled_default=False,
         device_class=SensorDeviceClass.ENUM,
         options=list(TANK_FILL.values()),
-        value_fn=lambda WasherDryer: TANK_FILL[
-            WasherDryer.get_attribute("WashCavity_OpStatusBulkDispense1Level")
-        ],
+        value_fn=lambda WasherDryer: TANK_FILL.get(
+            WasherDryer.get_attribute("WashCavity_OpStatusBulkDispense1Level"), None
```

Comment:
```suggestion
            WasherDryer.get_attribute("WashCavity_OpStatusBulkDispense1Level")
```

Reply:
This is the whole point of the change - if WashCavity_OpStatusBulkDispense1Level is not in the list, return None as the sensor doesn't exist (rather than crash the entire integration).
Additionally, we disable the sensor by default, so the user can enable the sensor if their device supports it.  If they do enable it and it doesn't support it, the integration crashes rather than just showing unknown.

Reply:
`None` is the default return value of `dict.get`.